instant-xml-macros/src/lib.rs

```diff
@@ -1,185 +1,120 @@
 extern crate proc_macro;
 
+mod de;
+mod se;
+
+use std::collections::{BTreeSet, HashMap};
+
 use proc_macro2::TokenStream;
 use quote::quote;
-use std::collections::{BTreeSet, HashMap};
-use syn::{parse_macro_input, Lit, Meta, NestedMeta};
+use syn::parse_macro_input;
+use syn::{Lit, Meta, NestedMeta};
+
+use crate::se::Serializer;
 
 const XML: &str = "xml";
 
-enum FieldAttribute {
+pub(crate) enum FieldAttribute {
     Namespace(String),
     PrefixIdentifier(String),
 }
 
-struct Serializer {
```

Comment:
This diff should not be part of this PR anymore, right? Otherwise you merged in a strange way.

Reply:
This because you merged it back 😄, the #11 PR was pointing to this branch not `main`.

Reply:
Ahh, so you can you do one pointing at main? Then merge main into this branch after we merge it.

Reply:
Sure, it's here: #12